<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyGUI_Threded_Server</class>
 <widget class="QWidget" name="MyGUI_Threded_Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyGUI_Threded_Server</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="pbStartStop">
     <property name="text">
      <string>Star/Stop</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>81</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="5">
    <widget class="QLineEdit" name="leSend"/>
   </item>
   <item row="0" column="0">
    <widget class="QLineEdit" name="leIP">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="sbPort">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="prefix">
      <string>port </string>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>3000</number>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="pbSend">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QComboBox" name="comboBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>DataStream</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TextStream</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QPlainTextEdit" name="peLog"/>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="tbClear">
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>tbClear</sender>
   <signal>clicked()</signal>
   <receiver>peLog</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
